public/css/game.css

```diff
@@ -118,7 +118,9 @@
   border: 1px solid #e35f53;
   transition: all 0.5s;
 }
-
+.introjs-helperLayer {
```

Comment:
Please name css selectors in all small letters.

Reply:
Thanks for your feedback. This css class is for overwriting the default behavior of a package called `intro-js` on the game screen. We did not create the class, we simply overwrote what the class did. Unfortunately we can't actually rename this class, since its created in the package using javascript.